lints/lint_dnsname_wildcard_left_of_public_suffix.go

```diff
@@ -0,0 +1,54 @@
+package lints
+
+import (
+	"github.com/weppos/publicsuffix-go/publicsuffix"
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zlint/util"
+)
+
+type DNSNameWildcardLeftofPublicSuffix struct {
+	// Internal data here
+}
+
+func (l *DNSNameWildcardLeftofPublicSuffix) Initialize() error {
+	return nil
+}
+
+func (l *DNSNameWildcardLeftofPublicSuffix) CheckApplies(c *x509.Certificate) bool {
+	return util.IsSubscriberCert(c)
+}
+
+func wildcardLeftOfPublicSuffix(domain string) (bool, ResultStruct) {
+	parsedDomain, err := publicsuffix.Parse(domain)
+	if err != nil {
+		return true, ResultStruct{Result: NA}
+	}
+	if parsedDomain.TRD == "" {
+		if parsedDomain.SLD == "*" {
+			return true, ResultStruct{Result: Warn}
```

Comment:
Why is this a WARN? The lint name is `e_dnsname_wildcard_left_of_public_suffix`.

Reply:
good catch, it should be w_.